docs/README-agent-install.md

```diff
@@ -206,13 +216,176 @@ search mycompany.com
 EOF
 ~~~
 
+## <a name="env-gs">Puppet Agent Environment: open agent container</a>
+
+This section is only necessary if Puppet will run from the `open agent container`.
+
+#### Set Up NX-OS
+
+Download the `OAC` `oac.1.1.0.ova` file.
+
+* For Cisco Nexus 7xxx devices: [Download Link](https://software.cisco.com/download/release.html?i=!y&mdfid=283748960&softwareid=282088129&release=7.3%280%29D1%281%29&os=)
+
+* For Cisco Nexus 56xx and 60xx devices: [Download Link]**TODO**()
+
+Copy the `ova` file to the `bootflash:` device.
+
+~~~
+n7k# dir bootflash: | inc oac.1.0.0.ova
+   45424640    Feb 12 19:37:40 2016  oac.1.0.0.ova
+n7k# 
+~~~
+
+Use the `show virtual-service global` command to display available resources for the `OAC` Virtual Service.
+
+~~~
+n7k# show virtual-service global 
+
+Virtual Service Global State and Virtualization Limits:
+
+Infrastructure version : 1.9
+Total virtual services installed : 0
+Total virtual services activated : 0
+
+Machine types supported   : LXC
+Machine types disabled    : KVM
+
+Maximum VCPUs per virtual service : 1
+
+Resource virtualization limits:
+Name                        Quota    Committed    Available
+-----------------------------------------------------------------------
+system CPU (%)                  6            0            6
+memory (MB)                  2304            0         2304
+bootflash (MB)                600            0          600
+
+n7k# 
+~~~
+
+The recommended minimum values are currently:
+
+~~~bash
+  memory    : 256MB
+  bootflash : 400MB
+~~~
```

Comment:
I would do this the same way we did it for guestshell. Just do:

```
\~~~
 n7k# show virtual-service global 
 ...
 Resource virtualization limits:
 Name                        Quota    Committed    Available
 -----------------------------------------------------------------------
 system CPU (%)                  6            0            6
 memory (MB)                  2304            0         2304
 bootflash (MB)                600            0          600
\~~~
```

The recommended minimum values are currently:
 
```
\~~~bash
   memory    : 256MB
   bootflash : 400MB
\~~~
```